cranelift-codegen/src/ir/immediates.rs

```diff
@@ -316,15 +316,15 @@ impl FromStr for Uimm128 {
             Err("Expected a hexadecimal string, e.g. 0x1234")
         } else if s.len() % 2 != 0 {
             Err("Hexadecimal string must have an even number of digits")
-        } else if s.len() > 32 {
+        } else if s.len() > 34 {
```

Comment:
???

Reply:
2 characters for `0x` plus 32 characters (16 x 2) for the data bytes

Reply:
Got it.